partcad-cli/src/partcad_cli/click/commands/supply/quote.py

```
@@ -1,12 +1,13 @@
 import rich_click as click
-
 import asyncio
 import copy
+import json
+import sys
+
 from partcad.provider_data_cart import *
```

Ruff (0.8.2)

`from partcad.provider_data_cart import *` used; unable to detect undefined names

(F403)